esimakova1984/ShopperShopIL
# added test add product from product page

added steps for test "Add a product from the product page":

1. Open the store page:The store page is opened to begin the product selection process.

2. Verify the cart is empty before adding products:Before adding any products, the cart is checked to ensure it is empty.

3. Select a product from the store page:A product is selected from the store page, and its details (name) are logged.

4. Add the selected product to the cart:The selected product is added to the cart. The product name is retrieved and logged for verification.

5. Open the cart page:After adding the product, the cart page is opened to verify the contents of the cart.

6. Verify that the selected product is in the cart:The cart is checked to ensure that the selected product is present. The test verifies that the product’s name matches the one added, and that no unexpected products are present in the cart.
